src/Illuminate/Support/Collection.php

```diff
@@ -352,13 +352,15 @@ public function keys()
 	}
 
 	/**
-	* Get the last item from the collection.
-	*
-	* @return mixed|null
-	*/
-	public function last()
+	 * Get the last item from the collection.
+	 *
+	 * @param  callable  $callback
+	 * @param  mixed  $default
+	 * @return mixed|null
```

Comment:
`mixed|null` is wrong btw. `mixed` already contains `null`. Should just be `mixed`.